### ISSUE TITLE
Pretrained weights for ScanNet indoor segmentation without val

### ISSUE THREAD

**Issue by zarzarj** [NVIDIA/MinkowskiEngine#396]
Hi,

I believe the current model used in the indoor example has been trained on the train + val set, but I require a model trained on just the train set. I was wondering if you could make available the weights for a model trained using the ScanNet dataset which does not use the validation set.

Thank you in advance!
